spec/factories/user.rb

```diff
@@ -25,6 +25,12 @@
       eligible_manager { true }
     end
 
+    ##
+    # A user who is allowed to manage a project
+    factory :data_manager do
```

Comment:
We should probably just change project_manager to be named data_manager so we do not confuse things by having two names for the same concept.

Reply:
Thank you @carolyncole ! - I have made those requested changes.